R/ei_iter.R

```diff
@@ -152,24 +161,103 @@ ei_iter <- function(
     set.seed(seed)
 
     # Run 2x2 ei
-    # ADD TRY CATCH SO THAT IF PARALLELIZATION=TRUE THEN KILL CLUSTERS
-    utils::capture.output({
-      ei_out <-
-        suppressMessages(
-          purrr::lift(ei::ei)(
-            data = data,
-            formula = formula,
-            total = totals_col,
-            erho = erho,
-            simulate = FALSE,
-            # args_pass
-          )
+    # Calls to ei are wrapped in nested try-catches that try different erho
+    # parameters to get the function working. The underlying ei function is
+    # unstable and changing erho can break it.
+    tryCatch(
```

Comment:
I wonder if there's a way to do this without nested try catch statements. Ultimately it makes the code harder to read and is harder to work on top of. 

One alternative is to use a while/for loop, iterating over possible `erho` values. Then, exit the loop once it passes without error.